eventhubs-streamanalytics-eventhubs/tools/README.md

```diff
@@ -1,11 +1,46 @@
-# Testing End-To-End Latency
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure eventhubs 
+extensions:
+- platforms: dotnet
+---
 
-Get the Event Hub connection string created and displayed during the script execution. If you cannot find it, just grab the execution string for the `Listen` Shared Access Policy for the output eventhub.
+# Getting started on processing received message from Event Hubs #
 
+Azure eventhubs-streamanalytics-eventhubs sample for processing received message from Event Hubs -
```

Comment:
I'm not quite sure what this line means to communicate.  Is this the summary of what the sample is?   If so, maybe consider something like:


The `eventhubs-streamanalytics-eventhubs` sample demonstrates processing messages from Event Hubs.  The `EventReceiver` will:
  - Connect to an existing event hub.
  - Read events from all partitions using the `EventHubConsumerClient`.
  - Process each received message and output to a `.csv` file.

Reply:
Yes it is the summary. Resolved.